language_version_metadata/Python36_CV__2__metadata.json

```diff
@@ -4,7 +4,7 @@
         "processor": "CPU",
         "mlPackageLanguage": "PYTHON36_CV",
         "languageGroup": "CV",
-        "baseImage": "python36cv:v21.8.1-2",
+        "baseImage": "python36cv:v21.10.0-2",
```

Comment:
Make sure you update the SF with the right tag for CV.

Reply:
yes, @mohitseth-nitrr is changing to these tags

Reply:
@snvenkat  builds are in progress and here is the PR for review https://github.com/UiPath/ai-pkgmanager/pull/594/files